website/docs/foundations/accessibility/index.md

```diff
@@ -0,0 +1,61 @@
+
+## Titles and Labels
+
+### Page titles
+
+Additionally, teams should ensure that each page has a relevant page title. This is included in Ember applications by default, but older applications that have been updated over the years may need to manually add this in with [ember-page-title](https://github.com/ember-cli/ember-page-title).
+
+### Headings
+
+Headings (h1-h6) should follow a logical order and not skip any levels. This can be tricky in dynamic applications so watch out for this one.
+
+### Input labels
+
+All inputs should have associated labels. These are provided for in the design system form components, but if you build your own make sure to check for these, remembering that a `placeholder` attribute is not an acceptable substitute.
+
+### Accessible name
+
+Remember, not every element is allowed to use `aria-label`. Check the [element's role documentation](https://www.w3.org/TR/wai-aria/#role_definitions) to see what aria-attributes are permissible. Just remember, when in doubt, ask! As the saying goes, "no ARIA is better than bad ARIA!"
+
+## Focus Management
+
+There are some key points to remember about focus management:
+
+1. Focus should return from whence it came. If a button toggled a modal window, closing that modal window should return focus to the button.
+2. Trap the focus! Focus should remain in a modal or other popup when it is open. Try [ember-focus-trap](https://github.com/josemarluedke/ember-focus-trap).
+3. Focus order should match DOM order.
+
+## Automated Testing
+
+Product applications should endeavor to use the automated tooling for static and dynamic analysis that are available to them. Ember apps can use [ember-template-lint](https://github.com/ember-template-lint/ember-template-lint) for static analysis and [ember-a11y-testing](https://github.com/ember-a11y/ember-a11y-testing) for dynamic analysis. Additionally, consider adding HTML validation to your CI.
+
+## Manual Testing
+
+- *Keyboard support:* ensure that your application is keyboard navigable. Everything that you can do with a mouse should have an equivalent experience with a keyboard!
+- *Browser extensions:* there are a few browser extensions that can help you with accessibility. Check out Microsoft's [Accessibility Insights](https://accessibilityinsights.io/) and Deque's [aXe](https://www.deque.com/axe/browser-extensions/) are two popular extensions that can help you check for accessibility issues right in your browser!
+- *Safari and VoiceOver:* your development environment has a screen reader built in! Become familiar with VoiceOver use it to test your application with Safari, especially if you are creating a non-standard UI and want to make sure that it will be inclusive for our users.
```

Comment:
1. I wonder if these should be headings instead of emphasized text so they're more scannable? 
2. Sorry, I know this is lame to call out, but I'm not sure if these instances warrant the exclamation. 
3. Do we know if all of our consumers are on Mac? Do we have any folks using other operating systems? Can we add details for those too?
4. There are a few plugins we can add to this list for Figma, I'll send them over tomorrow.

Reply:
@heatherlarsen Ideally I'd like to make two tabs, one for designers and one for developers. WDYT?

Reply:
Yea, that makes sense to me.